kovri.md

```diff
@@ -1,64 +1,68 @@
 They do not know anything about what address a transaction is coming from, where it is going to, or what the amount is.
 
-However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information. 
+However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information.
 Alternatively, they could purposefully attempt to block her transaction from being broadcast to the rest of the network.
 
 So what can Alice do to mitigate these threats?
 
-Alice does not know which nodes log her IP address, so she can't simply avoid the malicious nodes.
-Her only choice is to hide her IP address.
-There are several ways to hide an IP address, including with Tor, a trusted VPN, or I2P.
-We will focus on I2P in today's video, since this is the network on which Kovri operates.
+Alice does not know which Bob logs her IP address, so she can't simply avoid the malicious Bobs.
+Her only choice is to hide her IP address from Bob.
+
+There are several ways for her to hide her IP address from Bob.
+She could use Tor or a trusted VPN; but these both have their vulnerable weaknesses.
+
+The Tor network has trusted Directory Authorities which give a handful of Tor node operators overreaching influence of network consensus.
+
+Trusted VPNs are also susceptible to trivial bandwidth correlation; making it easy for 3-letter agencies to de-anonymize her traffic.
+VPNs also adhere to local laws in order to stay in business; so Alice always runs the risk of having her user data given to a government whose policies she doesn't agree with.
+Trusted VPNs also don't protect Bob from Alice knowing who Bob is.
+
+It's all so complicated! Fortunately, Alice and Bob have a better option for protecting their privacy. Their option is Kovri.
 
-In a sentence, Kovri is a lightweight router compatible with cryptocurrencies and built for the I2P network.
-I2P stands for Invisible Internet Protocol.
-I2P is a layer of the internet that allows for all connections in the network to be private.
-In contrast with regular internet routing or TOR's onion routing, I2P utilizes what it refers to as garlic routing.
-Instead of connecting with participants directly or through TOR nodes, such as going directly to a website, all information is encrypted and relayed through other I2P routers in the network.
+Kovri is a free, decentralized, anonymity technology developed by Monero.
+Currently based on the Invisible Internet Project's open specifications, Kovri uses both garlic encryption and garlic routing to create a private, protected overlay-network across the internet.
+This overlay-network provides Alice and Bob with the ability to effectively hide their geographical location and internet IP address.
+Essentially, Kovri *covers* an application's internet traffic to make it anonymous within the network.
+A lightweight and security-focused router, Kovri is fully compatible with the I2P network.
```

Comment:
"Kovri is lightweight, security-focused, and fully compatible with the I2P network"

Reply:
Agreed.